package.json

```diff
@@ -13,12 +13,13 @@
     "build": "node_modules/.bin/parallel-webpack",
     "doc": "node_modules/.bin/jsdoc -c jsdoc-conf.json",
     "test": "node_modules/.bin/karma start --single-run --browsers=ChromeHeadless",
-    "test:all": "for pkg in core jquery jquery-file-upload; do node_modules/.bin/karma start --single-run --cloudinary.pkg=${pkg}; done",
-    "test:all:prod": "for pkg in core jquery jquery-file-upload; do node_modules/.bin/karma start --single-run --cloudinary.minified --cloudinary.pkg=${pkg}; done",
+    "test:all:unminified": "for pkg in core jquery jquery-file-upload; do node_modules/.bin/karma start --single-run --cloudinary.pkg=${pkg}; done",
```

Comment:
@maoznir 
Would change to:
test:pkgs - run tests on all packages
test:pkgs:minified - run on minified packages
test:pkgs:verify - verify packages consistency